packages/headless/src/controllers/core/generated-answer/headless-core-generated-answer.ts

```diff
@@ -78,7 +81,9 @@ export interface GeneratedAnswer extends Controller {
    * Sends feedback about why the generated answer was not relevant.
    * @param feedback - The feedback that the end user wishes to send.
    */
-  sendFeedback(feedback: GeneratedAnswerFeedback): void;
+  sendFeedback(
+    feedback: GeneratedAnswerFeedback | GeneratedAnswerFeedbackV2
+  ): void;
```

Comment:
With the v3, we will update the parameter `feedback` here to be only of type `GeneratedAnswerFeedbackV2` right?
In that case I think we should add a `TODO` comment here with the jira ticket in order to update that when we start the v3.